src/facade/reply_builder.cc

```diff
@@ -296,17 +297,26 @@ void RedisReplyBuilder::SendError(string_view str, string_view err_type) {
     err_type = str;
     if (err_type == kSyntaxErr)
       err_type = kSyntaxErrType;
+    if (err_type == kLoadingErr)
+      err_type = kLoadingErrType;
   }
 
   tl_facade_stats->reply_stats.err_count[err_type]++;
 
   if (str[0] == '-') {
     iovec v[] = {IoVec(str), IoVec(kCRLF)};
     Send(v, ABSL_ARRAYSIZE(v));
-  } else {
-    iovec v[] = {IoVec(kErrPref), IoVec(str), IoVec(kCRLF)};
+    return;
+  }
+
+  if (err_type == kLoadingErrType) {
+    iovec v[] = {IoVec(kLoadingPrefix), IoVec(str), IoVec(kCRLF)};
     Send(v, ABSL_ARRAYSIZE(v));
+    return;
   }
+
+  iovec v[] = {IoVec(kErrPref), IoVec(str), IoVec(kCRLF)};
+  Send(v, ABSL_ARRAYSIZE(v));
```

Comment:
The inconsistency is that we don't use error types for prefixes for other cases (I don't even remember why we have them) and use the prefix directly, see `kAuthRejected` or `kScriptNotFound`... so what would be the preferred way now 🙂

Reply:
yes I see. error types are used for stats, you might want to send a different error message but register it under the same stat name